karaf/init/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+        <artifactId>brooklyn-tosca-karaf</artifactId>
+        <version>0.10.0-SNAPSHOT</version> <!-- BROOKLYN_TOSCA_VERSION -->
+        <relativePath>../pom.xml</relativePath>
```

Comment:
I personally don't like us using `<relativePath>`. We should just build the modules in the right order, and have them all pushed to artifactory - so no need to declare the relativePath.